unittests/runtime/Metadata.cpp

```diff
@@ -249,7 +185,7 @@ TEST(Concurrent, ConcurrentList) {
   }
 
   // Check that the length of the list is correct.
-  EXPECT_EQ(ListLen, results.size() * numElem);
+  EXPECT_EQ(ListLen, 0 * numElem);
```

Comment:
This looks wrong. Why are we verifying the contents of List and then asserting that List is empty?